diff --git a/apple/internal/rule_support.bzl b/apple/internal/rule_support.bzl
--- a/apple/internal/rule_support.bzl
+++ b/apple/internal/rule_support.bzl
@@ -214,7 +214,7 @@ def _create_ios_application_rule(
             bundle_locations = _describe_bundle_locations(archive_relative = "Payload"),
             bundle_package_type = bundle_package_type.application,
-            deps_cfg = apple_common.multi_arch_split,
+            deps_cfg = None,
             has_alternate_icons = True,
             has_launch_images = True,
             has_settings_bundle = True,
@@ -239,7 +239,7 @@ def _create_ios_application_rule(
             bundle_locations = _describe_bundle_locations(archive_relative = "Payload"),
             bundle_package_type = bundle_package_type.application,
-            deps_cfg = apple_common.multi_arch_split,
+            deps_cfg = None,
             has_alternate_icons = True,
             has_launch_images = True,
             has_settings_bundle = True,
@@ -259,7 +259,7 @@ def _create_ios_application_rule(
             bundle_locations = _describe_bundle_locations(archive_relative = "Payload"),
             bundle_package_type = bundle_package_type.application,
-            deps_cfg = apple_common.multi_arch_split,
+            deps_cfg = None,
             has_alternate_icons = True,
             has_launch_images = True,
             has_settings_bundle = True,
@@ -281,7 +281,7 @@ def _create_ios_application_rule(
             bundle_locations = _describe_bundle_locations(archive_relative = "Payload"),
             bundle_package_type = bundle_package_type.application,
-            deps_cfg = apple_common.multi_arch_split,
+            deps_cfg = None,
             has_alternate_icons = True,
             has_launch_images = True,
             has_settings_bundle = True,
